console_backend/src/tabs/solution_tab/solution_position_tab.rs

```diff
@@ -396,8 +398,6 @@ impl SolutionPositionTab {
         let dops_fields = msg.fields();
         self.table
             .insert(DOPS_FLAGS, format!("0x{:<03x}", dops_fields.flags));
-        self.table
-            .insert(INS_STATUS, format!("0x{:<01x}", self.ins_status_flags));
```

Comment:
not quite sure what this was doing in handle dops,
should INS status be updated based on DOPS updates?

Reply:
@john-michaelburke any ideas?

@adrian-kong can look at the old console? Maybe some hints there.  I'm not sure why we'd handle this in DOPS either.

Reply:
Yeah purely an artifact of copy pasta:
```
https://github.com/swift-nav/piksi_tools/blob/7725f247b978124f3ee7d23d67a0995f3aaa6f24/piksi_tools/console/solution_view.py#L573
```